README.md

```diff
@@ -22,6 +22,7 @@ These variables are defined in [defaults/main.yml](./defaults/main.yml):
 
     openssh_configuration_match_blocks: {}
 
+`Include: /etc/ssh/sshd_config.d/*.conf` will always be added to the configuration if the OpenSSH server supports it (> 8.2).
```

Comment:
This is not necessary per se, I'd suggest adding a new variable `openssh_add_dotconfig_when_feasible` that would be defaulted to `false`.

Such a change has 2 effects: 
1. the default value keeps 100% backwards compatibility with the existing code
2. adds new conditional capability that can be switched on and off